helpers/GenerateResourcesAndImage.ps1

```diff
@@ -119,6 +121,8 @@ Function GenerateResourcesAndImage {
         [Parameter(Mandatory = $False)]
         [bool] $EnableHttpsTrafficOnly = $False,
         [Parameter(Mandatory = $False)]
+        [Switch] $OnErrorCleanup,
```

Comment:
This param should be set like [ValidateSet attribute](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7.2) and default value is `ask`. Available values you can find - https://www.packer.io/docs/commands/build

Reply:
Updated, changed the name to be more generic